### PR TITLE
Peer to peer network updates

### PR DESCRIPTION
**Pull request by trevelyan**

### CODE REVIEW

File: bundler/default/lib/saito/network.js [hunk 1]
```diff
@@ -392,7 +381,41 @@ console.log("peer does not receive txs... not sending");
...
+    for (let x = this.peers.length - 1; x >= 0; x--) {
+      this.peers[x].sendRequest(message, data);
+    }
+  }
+  sendRequestWithCallback(message, data = "", callback) {
+    for (let x = this.peers.length - 1; x >= 0; x--) {
+      this.peers[x].sendRequestWithCallback(message, data, callback);
+    }
+  }
+  sendPeerRequest(message, data = "", peer) {
```
Comment (escapedcat) [inline bundler/default/lib/saito/network.js:401]: This is all defined twice?

File: lib/saito/network.js [hunk 2]
```diff
@@ -392,7 +381,41 @@ console.log("peer does not receive txs... not sending");
   }
 
 
+  sendPeerRequest(message, data = "", peer) {
```
Comment (escapedcat) [inline lib/saito/network.js:384]: Same here, defined twice

File: bundler/default/lib/saito/network.js (hunk 1 above)
Comment (escapedcat) [reply bundler/default/lib/saito/network.js:401]: Shouldn't all the `bundler` stuff be in `.gitignore` and not even be committed?

Comment (escapedcat) [reply bundler/default/lib/saito/network.js:401]: Yes, `bundler` is in the `.gitignore`. Not sure hwo this could even be added.

**Comment by trevelyan** [comment]
Thank you Hannes.

Will get fix done and pushed.

**Comment by escapedcat** [comment]
eslint/prettier would have noticed automatically ;)

**Comment by trevelyan** [comment]
ok, I'm sold.

maybe we can do overview of this stuff in 2 weeks or so.

**Comment by escapedcat** [comment]
> ok, I'm sold

hahaha, this took exactly one year. Here, have a look:  https://github.com/SaitoTech/saito-lite/pull/145

>. maybe we can do overview of this stuff in 2 weeks or so.

Sure, let's!

**Pull request merged**